api/jobs.py

```diff
@@ -258,6 +259,25 @@ def next(self):
 
         return result
 
+    def reap_stale(self):
```

Comment:
Given that there's still a component called "reaper", probably a good idea to avoid use of the term elsewhere.